src/jobs/publish_docker.yml

```diff
@@ -31,19 +29,71 @@ parameters:
     description: Overrides the default release branch docker tag (latest).
     type: string
     default: "latest"
+  ecr_login:
+    description: If true then login to ECR and set the ECR_URL environment variable to the ECR repo base URL
+    type: boolean
+    default: false
+  aws_profile:
+    description: The AWS profile to be used
+    type: string
+    default: "default"
+  region:
+    description: The AWS region for the deployment
+    type: string
+    default: eu-west-1
+  unpack_targets:
+    description: If true then unpack targets tarball
+    type: boolean
+    default: false
+  cmd:
+    description: Docker commands to tag and push the image. By default tag with version and push to dockerhub.
+    type: string
+    default: |
+      docker load --input "<< parameters.path >>"
```

Comment:
I think that passing CircleCI parameters in the `parameters` section is not working as intended..
@h314to

Reply:
Maybe we want to read everything from env variables and instead ?